src/NServiceBus.RavenDB/Subscriptions/RavenDbSubscriptionSettingsExtensions.cs

```diff
@@ -73,8 +73,34 @@ public static PersistenceExtensions<RavenDBPersistence> CacheSubscriptionsFor(th
         /// <returns></returns>
         public static PersistenceExtensions<RavenDBPersistence> DisableSubscriptionVersioning(this PersistenceExtensions<RavenDBPersistence> cfg)
         {
-            cfg.GetSettings().Set(DisableSubscriptionVersioningKey, true);
+            SetLegacyVersionedSubscriptions(cfg, false);
             return cfg;
         }
+
+        /// <summary>
+        /// Continue to use legacy method of subscription versioning that includes the message assembly
+        /// major version in the subscription document lookup key. This option will be obsoleted in NServiceBus.RavenDB 6.0.0,
```

Comment:
Instead of having obsolete information here, would it make more sense to put that in the attribute's message property?

Reply:
Added the pertinent parts to the obsolete message as well.